lib/containers/docker.pm

```diff
@@ -0,0 +1,116 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Basic functions for testing docker
+# Maintainer: Anna Minou <anna.minou@suse.de>
+
+package containers::docker;
+use base "consoletest";
+use testapi;
+use utils;
+use strict;
+use warnings;
+use version_utils;
+
```

Comment:
If you 'export' your functions you can simply call them like `set_up()` instead of `containers::docker->(set_up);`.

```perl
our @EXPORT = qw(set_up build_img test_built_img);
```

Reply:
Also good hint, it is added too!